### PR TITLE
Update .env.sample

### PR DESCRIPTION
**Pull request by SimiHunjan**
Remove the name variable as it will no longer be used.

### CODE REVIEW

**Comment by SimiHunjan** [comment]
Fixes #7

**Review by theekrystallee — APPROVED** [review]
lgtm

File: transfer-hbar/script-transfer-hbar.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ async function scriptTransferHbar() {
     const transferJsonAccountTransfersFinalAmounts = transferJsonAccountTransfers
         ?.map((obj) => Hbar.from(obj.amount, HbarUnit.Tinybar).toString(HbarUnit.Hbar));
     logger.log(
-        'The debit and credit amounts of the transfer transaction:\n',
+        'The debit, credit, and transaction fee amounts of the transfer transaction:\n',
```
Comment (bguiz) [inline transfer-hbar/script-transfer-hbar.js:118]: This change is replicated in #9
See https://github.com/hedera-dev/hello-future-world-js/pull/10/files#r1701160360

File: .env.sample [hunk 2]
```diff
@@ -3,9 +3,6 @@
 # as this file is stored as plain text on disk,
 # and is therefore not secure enough.
 
-# Name
-YOUR_NAME=
-
```
Comment (bguiz) [inline .env.sample:8]: This change needs to be made upstream, in https://github.com/hedera-dev/hedera-tutorial-demo-base-template/blob/main/.env.sample
and then replicated here

Comment (SimiHunjan) [reply .env.sample:8]: done https://github.com/hedera-dev/hedera-tutorial-demo-base-template/pull/1

File: transfer-hbar/script-transfer-hbar.js (hunk 1 above)
Comment (SimiHunjan) [reply transfer-hbar/script-transfer-hbar.js:118]: once this is merged the second PR will automatically update with not modifying this no?

**Comment by SimiHunjan** [comment]
I will just close this PR and #10  should contain all the updates in one go from a feat branch instead of main

**Pull request closed without merging**